ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -193,6 +193,22 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test-disconnected
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-sno
+- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
```

Comment:
We need add extra pre-steps before upgrade for this profile in following pr, right?

Reply:
I think that depends on if there is some different CRs are introduced between 4.13 and 4.14. If there is no difference by chance, then maybe do not need a pre-upgrade step for the dedicated CR operation. cc @jianping-shu to confirm.

Reply:
But for a long-term gold, we should have the pre-upgrade step to check the CR difference between versions, so that ensure the required CRs is provided before upgrade.

Reply:
With `--included` option, we can run the cmd before upgrade to check the update, and that's why we add one more upgrade case for it. 
My concern is that this new added upgrade job will fail until we add step6&step7 in the [case](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-66839). @jianping-shu pls correct me if my understanding is wrong. 
As for the integration for step6&7, I'm still thinking about the best way for the case automation, just as suggested in automation meeting just now.

Reply:
Let me double check the 4.13 CRs vs. 4.14 ones and update later

Reply:
Take an example, if I understand right, source cluster has 5 CRs, target cluster requires 6 CRs, once that, before upgrade, we definitely need to create one more that is missing in the source cluster whatever from the regular team ci view or the OTA automation view.

Reply:
Yeah, if we definitely need one more step for upgrade here, how about updating `f28` to `f365` before we support it actually? and re-update it back once integrating the step in ci. Or else, the upgrade failure will be caught by the alert and we need do failure analysis:)

Reply:
That's right!
I just double checked the 4.14 aws CRs changes vs. 4.13.
There are permission added for 2 CRs i.e. cluster-ingress-operator and cluster-storage-operator in 4.14. You can find the details in Column K in [CredentialsRequest tracking](https://docs.google.com/spreadsheets/d/1K4lZpA9hp9uW5XpieUJeqt8-K8Ff5fVPcFyH6W943Uk/edit#gid=0)
According to past experiences, the upgrade might be successful without pre-steps unless the added permission is mandatory for the operator startup.

As to upgradeable-to annotation, its logic should be already done in function cco_annotation for upgrade.

Reply:
thanks for the confirm, then we could have a test on the upgrade once a green build available.

Reply:
agree, we can have a test to see the results.